apps/web/src/server/appeals/inspector/__tests__/__snapshots__/inspector.test.js.snap

```diff
@@ -518,24 +518,23 @@ exports[`inspector GET /appeals/:appealId/book-site-visit should render a page f
                                 <div class=\\"govuk-form-group\\">
                                     <label class=\\"govuk-label govuk-date-input__label\\" for=\\"siteVisitDate\\">Day</label>
                                     <input class=\\"govuk-input govuk-date-input__input govuk-input--width-2 \\"
```

Comment:
changes in the appeals scope are a side effect of updating the govuk-frontend package